pixeltable/env.py

```diff
@@ -339,17 +349,36 @@ def _set_up(self, echo: bool = False, reinit_db: bool = False) -> None:
         http_logger.propagate = False
 
         self.clear_tmp_dir()
-
         self._db_name = os.environ.get('PIXELTABLE_DB', 'pixeltable')
-        self._pgdata_dir = Path(os.environ.get('PIXELTABLE_PGDATA', str(Config.get().home / 'pgdata')))
 
-        # cleanup_mode=None will leave the postgres process running after Python exits
-        # cleanup_mode='stop' will terminate the postgres process when Python exits
-        # On Windows, we need cleanup_mode='stop' because child processes are killed automatically when the parent
-        # process (such as Terminal or VSCode) exits, potentially leaving it in an unusable state.
-        cleanup_mode = 'stop' if platform.system() == 'Windows' else None
-        self._db_server = pixeltable_pgserver.get_server(self._pgdata_dir, cleanup_mode=cleanup_mode)
-        self._db_url = self._db_server.get_uri(database=self._db_name, driver='psycopg')
+        db_connect_str = config.get_string_value('DB_CONNECT_STR')
+        db_default_connect_str = config.get_string_value('DEFAULT_DB_CONNECT_STR')
```

Comment:
What's the relationship between `DEFAULT_DB_CONNECT_STR` and `DB_CONNECT_STR`? Some code comments that explain what's going on would be helpful.